exchanges/zb/zb_test.go

```diff
@@ -183,7 +183,7 @@ func setFeeBuilder() *exchange.FeeBuilder {
 func TestGetFeeByTypeOfflineTradeFee(t *testing.T) {
 	var feeBuilder = setFeeBuilder()
 	z.GetFeeByType(feeBuilder)
-	if !areTestAPIKeysSet() {
+	if !z.ValidateAPICredentials() {
```

Comment:
When we call this, we are calling UpdateAccountInfo so it might be better to reside in testmain and only call it once to validate the keys. It's not a huge issue but we are doubling our outbound auth requests in testing.

Reply:
Could you point me to the code you're referring to here? `ValidateAPICredentials` is a base exchange function which looks at config values

Reply:
Doh! That's my wrong, you are correct. I was looking at the Wrapper func ValidateCredentials. My mistake!